setup.py

```diff
@@ -874,17 +864,16 @@ def _postinstall(self):
             raise RuntimeError(f"Can't find '{filename}'")
         # As of setuptools>=74.0.0, we no longer need to
         # be concerned about distutils calling win32api
-        subprocess.Popen(
```

Comment:
I don't think this replaces https://github.com/mhammond/pywin32/pull/2392
That other PR also tests uninstalling
In fact both together should be better as it would test the install under 2 different contexts, *and* test a re-install.